mono/mini/mini-amd64.c

```diff
@@ -3833,14 +3833,16 @@ emit_setup_lmf (MonoCompile *cfg, guint8 *code, gint32 lmf_offset, int cfa_offse
 
 #ifdef TARGET_WIN32
 
+#define TEB_LAST_ERROR_SLOT 0x30
 #define TEB_LAST_ERROR_OFFSET 0x068
 
 static guint8*
 emit_get_last_error (guint8* code, int dreg)
 {
 	/* Threads last error value is located in TEB_LAST_ERROR_OFFSET. */
 	x86_prefix (code, X86_GS_PREFIX);
-	amd64_mov_reg_membase (code, dreg, TEB_LAST_ERROR_OFFSET, 0, sizeof (guint32));
+	amd64_mov_reg_mem (code, dreg, TEB_LAST_ERROR_SLOT, sizeof (gpointer));
```

Comment:
Why this change? I see it matches GetLastError, but it looked correct and more optimal before.